src/ML-METATOMIC/pair_metatomic.cpp

```diff
@@ -718,7 +718,7 @@ void PairMetatomic::compute(int eflag, int vflag) {
 
             error->warning(FLERR,
                 "The uncertainty on atomic energies for {} are larger than "
-                "the threshold of {}. Be careful when analyzing the results, "
+                "the threshold of {} eV/atom. Be careful when analyzing the results, "
```

Comment:
no, the threshold value is in whatever unit is being used by lammps, not always eV